build.sh

```diff
@@ -19,7 +19,7 @@ export GOPATH=${PWD}/gopath
 mkdir -p "${PWD}/bin"
 
 echo "Building plugins"
-PLUGINS="plugins/meta/* plugins/main/* plugins/ipam/* plugins/sample"
+PLUGINS="plugins/meta/* plugins/main/* plugins/ipam/* plugins/sample plugins/tc/*"
```

Comment:
cool.   you'll need to add it to [`plugins/linux_only.txt`](https://github.com/containernetworking/plugins/blob/master/plugins/linux_only.txt) also to fix the AppVeyor (Windows CI) build.